packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -130,6 +131,7 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     log.info('> Cancelling download. Closing payment channels, and then removing torrent');
     const torrent = this.torrents.find(t => t.infoHash === infoHash);
     if (torrent) {
+      torrent.pause();
```

Comment:
Maybe add a bit of explanation, since the intended effect is a bit subtle?
```suggestion
      // I am only allowed to close the channel on my turn.
      // If I don't pause the torrent, then I will continue to make payments, meaning the call to 
      // this.closeChannels would race the `makePayment` function, and one of them would fail with a
      // "Not my turn" error.
      torrent.pause();
```